internal/apiserver/server.go

```diff
@@ -476,9 +476,15 @@ func (as *apiServer) createMuxRouter(ctx context.Context, o orchestrator.Orchest
 				Methods(route.Method)
 		}
 	}
+	r.HandleFunc("/api/v1/namespaces/{ns}/apis/{apiName}/ui", func(rw http.ResponseWriter, req *http.Request) {
```

Comment:
I think `/api` is what we've used at the root for the UI version, so we should probably just use that.
Not sure how people feel about `/apispec` for the yaml/JSON spec, or whether we should have `/spec.yaml` `/spec.json` or if people know of a better convention.

Reply:
I was unsure if this is too many instances of "api" in a single URL 🙃
http://localhost:5000/api/v1/namespaces/default/apis/simple-storage/api

We could consider dropping the last path segment, and doing something like:
UI: http://localhost:5000/api/v1/namespaces/default/apis/simple-storage
JSON: http://localhost:5000/api/v1/namespaces/default/apis/simple-storage.json

That is, unless the first URL is already a fetch-by-name for the FF ContractAPI object...

We could also consider a query parameter, such as:
FireFly object: http://localhost:5000/api/v1/namespaces/default/apis/simple-storage
UI: http://localhost:5000/api/v1/namespaces/default/apis/simple-storage?format=ui
JSON: http://localhost:5000/api/v1/namespaces/default/apis/simple-storage?format=json

Reply:
NestJS uses `/api` and `/api-json`, just to throw out another convention we've used in a few projects.

Reply:
I agree with Peter using `/api` over `/ui`. I also agree with Andrew that it's a lot of `api`s all in one URL. I'm okay with that though.

Reply:
Seems like the path of least resistance is to just copy what is in FireFly base (ie `.../api` for the Swagger UI, and `.../api/swagger.json` for the JSON)